scripts/verify

```diff
@@ -111,6 +111,14 @@ function print_summary() {
     echo "$deployer_errors" >> "$error_summary_path"
   fi
 
+  if [[ -n "$validation_errors" ]]; then
+    echo "=== SCHEMA VALIDATION WARNING ==="
+    echo "WARNING: Schema is incompatible with the latest deployer, would fail with:"
```

Comment:
It occurred to me that we should _additionally_ update `deploy_with_tests.sh` to also call schema validation. This should be safe because it's only invoked during `verify`, and if the partner doesn't update their deployer, the new validations wouldn't apply (i.e. verification still passes).

Reply:
Both `deploy.sh` [1] and `deploy_with_tests.sh` [2] already call `Schema.verify()` via `expand_config.py` [3], which. That's why the `schema.yaml` in our tests *had* to be updated despite no other new validation call additions, since the integration tests use the updated base deployer.

Happy to make other changes in a follow-up if this doesn't meet the concern.

[1] https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/master/marketplace/deployer_util/deploy.sh#L60
[2] https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/master/marketplace/deployer_util/deploy_with_tests.sh#L67
[3] https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/master/marketplace/deployer_util/expand_config.py#L69